### DISCUSSION TITLE
Test with Jest/Ava same import error

### DISCUSSION THREAD

**Discussion by giovanni9793** [panva/jose#164] [Q&A]
Like other people, I run into the problem of the import in jest, and the possible solutions. One of the solution resolve the problem, but added one other ("ReferenceError: TextEncoder is not defined") that the suggestions can't resolve. I understood that the problem of the import was related only to jest, so I decided to change environment and use ava, but the same problem (with the same solution and following problem) is also there, so what I'm doing wrong?
Last version of jose, ava and node.

jest config:
```
module.exports = {
  coverageReporters: [
    'text',
    'cobertura',
  ],
  moduleFileExtensions: ['js', 'ts', 'json'],
  moduleNameMapper: {
    '^jose/(.*)$': '<rootDir>/node_modules/jose/dist/node/cjs/$1',
  },
  modulePathIgnorePatterns: ['/test/0excluded/*', '/test/ava/*'],
  preset: 'ts-jest',
  testEnvironment: 'node',
  testMatch: [
    '**/?(*.)+(spec|test).ts?(x)',
  ],
  transform: {
    '^.+\\.ts?$': 'ts-jest',
  },
}
```

ava config:
```
export default {
  files: [
    'test/ava/**/*',
    '!test/0excluded/**/*',
  ],
  verbose: true,
  nodeArguments: ['--trace-deprecation', '--napi-modules'],
  nonSemVerExperiments: {
    configurableModuleFormat: true,
  },
  typescript: {
    rewritePaths: {
      'src/': 'dist/',
    },
  },
  require: [
    'ts-node/register',
    './node_modules/jose/dist/node/cjs/jwt/verify.js',
  ],
}

```

**Comment by panva** [comment]
Duplicate of #105 where you also have everything [answered](https://github.com/panva/jose/discussions/105#discussioncomment-210269).
